addons/longhorn/1.1.1/host-preflight.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: troubleshoot.sh/v1beta2
+kind: HostPreflight
+metadata:
+  name: longhorn
+spec:
+  collectors:
+  - diskUsage:
+      collectorName: "Longhorn Disk Usage"
+      path: /var/lib/longhorn
+
+  analyzers:
+  - diskUsage:
+      checkName: "Longhorn Disk Usage"
+      collectorName: "Longhorn Disk Usage"
+      exclude: '{{kurl and .Installer.Spec.Longhorn.Version (not .IsUpgrade) | not }}' # only run if there is a longhorn version set and this is not an upgrade
```

Comment:
I think the check for Longhorn version is redundant in here

Reply:
fixed it before the review, funnily enough
Race conditions!